java/datastructures/lib/src/main/java/codechallenges/linkedlist/Node.java

```diff
@@ -0,0 +1,11 @@
+package codechallenges.linkedlist;
+
+public class Node extends datastructures.linkedlist.Node{
+  public int value;
```

Comment:
You don't need this Node class. Instead, import the Node class from your datastructures directory into LinkedListChallenges